examples/analytics/emr-on-eks/main.tf

```diff
@@ -118,52 +80,77 @@ module "eks_blueprints" {
   source = "../../../modules/kubernetes-addons"
 
   eks_cluster_id = module.eks_blueprints.eks_cluster_id
-  #K8s Add-ons
+
+  # Add-ons
   enable_metrics_server     = true
   enable_cluster_autoscaler = true
 
-  #---------------------------------------
-  # PROMETHEUS and Amazon Prometheus Config
-  #---------------------------------------
   # Amazon Prometheus Configuration to integrate with Prometheus Server Add-on
   enable_amazon_prometheus             = true
   amazon_prometheus_workspace_endpoint = module.eks_blueprints.amazon_prometheus_workspace_endpoint
 
-  # Enabling Prometheus Server Add-on
   enable_prometheus = true
-  # Optional Map value
   prometheus_helm_config = {
-    name       = "prometheus"                                         # (Required) Release name.
-    repository = "https://prometheus-community.github.io/helm-charts" # (Optional) Repository URL where to locate the requested chart.
-    chart      = "prometheus"                                         # (Required) Chart name to be installed.
+    name       = "prometheus"
+    repository = "https://prometheus-community.github.io/helm-charts"
+    chart      = "prometheus"
     version    = "15.3.0"
-    # (Optional) Specify the exact chart version to install.
-    namespace = "prometheus" # (Optional) The namespace to install the release into.
+    namespace  = "prometheus"
     values = [templatefile("${path.module}/helm_values/prometheus-values.yaml", {
       operating_system = "linux"
     })]
   }
-  #---------------------------------------
-  # Vertical Pod Autoscaling
-  #---------------------------------------
-  enable_vpa = true
 
+  enable_vpa = true
   vpa_helm_config = {
-    name       = "vpa"                                 # (Required) Release name.
-    repository = "https://charts.fairwinds.com/stable" # (Optional) Repository URL where to locate the requested chart.
-    chart      = "vpa"                                 # (Required) Chart name to be installed.
-    version    = "1.0.0"                               # (Optional) Specify the exact chart version to install
-    namespace  = "vpa"                                 # (Optional) The namespace to install the release into.
+    name       = "vpa"
+    repository = "https://charts.fairwinds.com/stable"
+    chart      = "vpa"
+    version    = "1.0.0"
+    namespace  = "vpa"
     values     = [templatefile("${path.module}/helm_values/vpa-values.yaml", {})]
   }
 
+  tags = local.tags
+
   depends_on = [module.eks_blueprints.managed_node_groups]
 }
+
+#---------------------------------------------------------------
+# Supporting Resources
+#---------------------------------------------------------------
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = local.name
+  cidr = "10.0.0.0/16"
+
+  azs             = ["${local.region}a", "${local.region}b", "${local.region}c"]
+  private_subnets = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
```

Comment:
Why change from the previous approach?

Reply:
Just trying to reduce the amount of configs users need to copy+paste to replicate the blueprint in their current codebase. Functionally, it achieves the same thing but we're focusing more on the blueprints and less on VPC (which I assume will already exist for most users so lets keep that out of the "signal"). Does that answer your question?

### Before
![image](https://user-images.githubusercontent.com/10913471/168859597-d541b89d-4dff-44c2-a34b-8104b1848993.png)

### After
![image](https://user-images.githubusercontent.com/10913471/168859529-a3f23b7d-cc8d-4b9a-8c1b-8eb859554baf.png)

Reply:
Gotcha...more specifically I'm referring to the azs and private subnets props. I.E. https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/complete-kubernetes-addons/main.tf#L50

Reply:
I will revert